packages/discount-code-importer/package.json

```diff
@@ -20,8 +20,20 @@
   "contributors": [
     "Williams Omayuku <williams.omayuku@commercetools.com>"
   ],
-  "main": "lib/index.js",
+  "main": "./lib/index.js",
```

Comment:
Yes you have, and no it doesn't... Just for consistency; it's about 50:50 in the `monorepo` packages for and against